.travis.yml

```diff
@@ -28,7 +26,7 @@ matrix:
       env: SOLR_VERSION="4.10.4" TEST_CONFIG="phpunit-integration-legacy-solr.xml" CONTAINER_TEST_CONFIG="tests/integration_legacy_solr_single_core.yml" SOLR_CONFS="eZ/Publish/Core/Search/Solr/Content/Resources/schema.xml"
 # 5.5
     - php: 5.5
-      env: TEST_CONFIG="phpunit.xml" SYMFONY_VERSION="~2.8@dev"
+      env: TEST_CONFIG="phpunit.xml"
```

Comment:
Maybe we could keep `~2.8@dev`, but **not** in `allowed_failures` section.
It's important that we know we're still compatible with it IMHO.